packages/internals/src/errorReporting.ts

```diff
@@ -17,7 +17,7 @@ export interface CreateErrorReportInput {
   kind: ErrorKind
   liftRequest?: string
   operatingSystem: string
-  platform: string
+  platform: string // TODO: maybe rename to binaryTarget in https://github.com/prisma/error-handling-backend
```

Comment:
```suggestion
  // We wanted to rename this to `binaryTarget` but we had to skip the rename here
  // because we didn't want to create more work
  // as the error handling backend would need to be updated and re-deployed
  // https://github.com/prisma/error-handling-backend
  platform: string
```